images/Dockerfile

```diff
@@ -0,0 +1,21 @@
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+ARG RELEASE_VER=4.6
```

Comment:
Please set this to the latest version.

We will have to write a release pipeline for deploying different versions of this Docker image. As ppc64le is also involved we can look at Travis jobs during `tag` events. @bpradipt

Reply:
Updated RELEASE_VER to 4.7